webargs/fields.py

```diff
@@ -36,3 +36,33 @@ def __init__(self, nested, *args, **kwargs):
         if isinstance(nested, dict):
             nested = argmap2schema(nested)
         super(Nested, self).__init__(nested, *args, **kwargs)
+
+class DelimitedList(ma.fields.List):
+    """Same as `marshmallow.fields.List`, except can load from either a list or
+    a delimited string (e.g. "foo,bar,baz").
+
+    :param Field cls_or_instance: A field class or instance.
+    :param str delimiter: Delimiter between values.
+    :param bool dump_string: Dump values to string.
```

Comment:
While I can see the reason for this naming, I think I'd rather this be `as_string` if only for consistency with `fields.Number`.